crazyflie/scripts/crazyflie_server.py

```diff
@@ -944,6 +949,21 @@ def _cmd_hover_changed(self, msg, uri=""):
         self.get_logger().info(
             f"{uri}: Received hover topic {vx} {vy} {yawrate} {z}")
 
+    def _cmd_full_state_changed(self, msg, uri=""):
+        """
+        Topic update callback to full state cmd topic
+        """
+        pos = [msg.pose.position.x, msg.pose.position.y, msg.pose.position.z]
+        vel = [msg.twist.linear.x, msg.twist.linear.y, msg.twist.linear.z]
+        acc = [msg.acc.x, msg.acc.y, msg.acc.z]
+        q = [msg.pose.orientation.x, msg.pose.orientation.y, msg.pose.orientation.z, msg.pose.orientation.w]
+        roll_rate = msg.twist.angular.x
+        pitch_rate =  msg.twist.angular.y
+        yaw_rate = msg.twist.angular.z
+      
+        self.swarm._cfs[uri].cf.commander.send_full_state_setpoint(pos, vel, acc, q, roll_rate, pitch_rate, yaw_rate)
+        self.get_logger().info(f"{uri}: Received full state topic {pos} {vel} {acc} {q}, {roll_rate}, {pitch_rate}, {yaw_rate}")
```

Comment:
This is usually called at a very high rate (100 Hz or so), so I wouldn't use any logging here. At least in the cpp backend we only do it for service calls.

Reply:
got it, I removed it together with the twist hover logging as well